.vscode/launch.json

```diff
@@ -1,266 +1,208 @@
-                    "--no-run",
-                    "--bin=puppet-chain",
-                    "--package=puppet-chain"
-                ],
-                "filter": {
-                    "name": "puppet-chain",
-                    "kind": "bin"
-                }
-            },
-            "args": [],
-            "cwd": "${workspaceFolder}"
-        },
-        {
-            "type": "lldb",
-            "request": "launch",
-            "name": "benchmark 'marf_bench'",
-            "cargo": {
-                "args": [
-                    "test",
-                    "--no-run",
-                    "--bench=marf_bench",
-                    "--package=blockstack-core"
-                ],
-                "filter": {
-                    "name": "marf_bench",
-                    "kind": "bench"
-                }
-            },
-            "args": [],
-            "cwd": "${workspaceFolder}"
-        },
-        {
-            "type": "lldb",
-            "request": "launch",
-            "name": "benchmark 'large_contract_bench'",
-            "cargo": {
-                "args": [
-                    "test",
-                    "--no-run",
-                    "--bench=large_contract_bench",
-                    "--package=blockstack-core"
-                ],
-                "filter": {
-                    "name": "large_contract_bench",
-                    "kind": "bench"
-                }
-            },
-            "args": [],
-            "cwd": "${workspaceFolder}"
-        },
-        {
-            "type": "lldb",
-            "request": "launch",
-            "name": "benchmark 'block_limits'",
-            "cargo": {
-                "args": [
-                    "test",
-                    "--no-run",
-                    "--bench=block_limits",
-                    "--package=blockstack-core"
-                ],
-                "filter": {
-                    "name": "block_limits",
-                    "kind": "bench"
-                }
-            },
-            "args": [],
-            "cwd": "${workspaceFolder}"
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "executable 'blockstack-core'",
+      "cargo": {
+        "args": ["build", "--bin=stacks-inspect", "--package=blockstack-core"],
+        "filter": {
+          "name": "stacks-inspect",
+          "kind": "bin"
         }
-    ]
+      },
+      "args": [],
+      "cwd": "${workspaceFolder}"
+    },
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "executable 'clarity-cli'",
+      "cargo": {
+        "args": ["build", "--bin=clarity-cli", "--package=blockstack-core"],
+        "filter": {
+          "name": "clarity-cli",
+          "kind": "bin"
+        }
+      },
+      "args": [],
+      "cwd": "${workspaceFolder}"
+    },
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "executable 'blockstack-cli'",
```

Comment:
Isn't this called `stacks-inspect` now?

Reply:
Not sure - this is isn't something I changed in my branch, but I can update that file in this PR if it needs to be changed.